src/utils/network.h

```diff
@@ -102,15 +102,15 @@ static inline void *pg_utils_get_l3(struct rte_mbuf *pkt)
 	return rte_pktmbuf_mtod(pkt, char *) + pkt->l2_len;
 }
 
-static inline int pg_utils_get_ipv6_l4(struct rte_mbuf *pkt, uint8_t *ip_type,
-				       uint8_t **ip_payload)
+static inline int pg_utils_iter_l3(struct rte_mbuf *pkt, uint8_t *ip_type,
```

Comment:
s/pg_utils_iter_l3/pg_utils_iter_ipv6_l3/
or pg_utils_get_ipv6_l4_internal
but this function work only with ipv6, so please don't remove this information from the name